### PR TITLE
Fixes #10744: Document correctly memory usage 

### PR DESCRIPTION
**Pull request by ncharles**
https://www.rudder-project.org/redmine/issues/10744

### CODE REVIEW

File: 10_installation/05_requirements/22_architecture_root_server.txt [hunk 1]
```diff
@@ -22,11 +22,12 @@ Rudder Server runs on both 32 (if available) and 64 bit versions of every suppor
 
 ===== Memory
 
-The required amount of RAM mainly depends on the number of managed nodes. A general rule for the minimal value is:
+The required amount of RAM mainly depends on the number of managed nodes. A general rule for the minimal value on a monolithic server is:
```
Comment (fanf) [inline 10_installation/05_requirements/22_architecture_root_server.txt]: stand-alone installation?

**Comment by ncharles** [comment]
Commit modified

**Comment by ncharles** [comment]
Commit modified

**Comment by ncharles** [comment]
OK, merging this PR

**Comment by ncharles** [comment]
OK, merging this PR

**Pull request merged**